src/app/helptext/storage/volumes/volume-status.ts

```diff
@@ -0,0 +1,73 @@
+import { T } from '../../../translate-marker';
+
+export default {
+    dialogFormFields: {
+        disk: {
+            placeholder: T('Member disk'),
+            tooltip: T(''),
```

Comment:
Rough text suggestion:

Choose a new disk for the pool. To protect any existing data, adding the selected disk is stopped when the disk is already in use or has partitions present.